Makefile

```diff
@@ -25,14 +25,23 @@ bootstrap:
 	pip install -r requirements.txt
 	pip install -r requirements-dev.txt
 	pip install -r requirements-tests.txt
+	pip install virtualenv
```

Comment:
this doesn't make sense here, the target already assumes in L21 we're running in virtualenv

Reply:
or is this a module used inside virtualenv?

Reply:
Yes, we need to make a separate virtualenv and this is a simple way to do that without mucking around with the containing venv context. We need a separate virtualenv because we need to try importing without the test or dev requirements, just the ones declared in setup.py